source/sync.txt

```diff
@@ -182,3 +142,43 @@ Explore the {+sync-short+} Documentation
       :ref:`sync-production-load-testing` guide, then get your app ready to ship
       with the :ref:`sync-production-checklist`.
 
+
+.. kicker:: More Info
+
+End-to-End Security
+-------------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: On Device
+      :icon: /images/icons/Technical_REALM_Mobile3x.png
+      :icon-alt: An icon depicting a mobile app
+
+      Encrypt PII in Realm Database files with the encryption APIs.
```

Comment:
Nit: does this have to be PII? Could it instead be "sensitive data"? Not all PII requires encryption, and I think this may also be a pretty US-centric concept.